### PR TITLE
[@types/stripe] Added missing webhook endpoints

### PR DESCRIPTION
**Pull request by Findeton**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://stripe.com/docs/api/webhook_endpoints?lang=node
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/stripe/index.d.ts [hunk 1]
```diff
@@ -9469,6 +9621,115 @@ declare namespace Stripe {
         }
     }
 
+    namespace webhookEndpoints {
+        interface IWebhookCreateOptions {
+            /**
+             * The URL of the webhook endpoint
+             */
+
```
Comment (pushplay) [inline types/stripe/index.d.ts]: Unnecessary extra line between the comment and the member.

File: types/stripe/index.d.ts [hunk 2]
```diff
@@ -9469,6 +9621,115 @@ declare namespace Stripe {
...
+            /**
+             * Value is 'webhook_endpoint'
+             */
+            object: "webhook_endpoint";
+
+            id: string;
+            /**
+             * The Stripe API version used to render data.
+             */
+
```
Comment (pushplay) [inline types/stripe/index.d.ts]: Just clean up the newlines here.

**Review by pushplay — CHANGES_REQUESTED** [review]
This looks really good.  Thanks for the tests.  Just a couple style consistency items to fix and it's good to go.

**Review by pushplay — APPROVED** [review]

**Comment by PranavSenthilnathan** [comment]
Congratulations on your first DefinitelyTyped contribution! Thank you for being a part of the community!

**Pull request merged**